src/jobs/publish.yml

```diff
@@ -98,6 +103,11 @@ parameters:
     type: boolean
     default: true
 
+  update-description:
+    description: Update the image description on Docker Hub?
+    type: boolean
+    default: true
```

Comment:
default false to be backward compatible and inline with default behavior of docker hub push